model/client4.go

```diff
@@ -2912,6 +2916,16 @@ func (c *Client4) TestEmail(config *Config) (bool, *Response) {
 	return CheckStatusOK(r), BuildResponse(r)
 }
 
+// TestSiteURL will attempt to connect to the configured SMTP server.
```

Comment:
This seemed to be not accurate since  I don't see it being tested against the configured SMTP server.